lang/llvm-16/Portfile

```diff
@@ -506,19 +506,13 @@ if {${subport} eq "clang-${llvm_version}"} {
     variant libstdcxx description {-stdlib=libstdc++_macports searches for MacPorts libstdc++} {
 
         patchfiles-append 0005-clang-support-macports-libstdcxx.patch
+        depends_run-append  port:libstdcxx_clang_fix
 
         post-patch {
-            if { ${configure.build_arch} eq "arm64" } {
-                # see https://github.com/macports/macports-ports/commit/be1e11a368f672d927a7bdb381f2fa71a79ba483
-                set gcc_arch    aarch64
-            } else {
-                set gcc_arch    ${configure.build_arch}
-            }
-
```

Comment:
Not sure I see why you can just remove this ? If the comment in

https://github.com/macports/macports-ports/commit/be1e11a368f672d927a7bdb381f2fa71a79ba483

that aarch64 should be used on ARM no longer valid for some reason ?

Reply:
As @michaelld committed the above he should comment on if it is now valid to just remove this...

Reply:
https://github.com/macports/macports-ports/commit/be1e11a368f672d927a7bdb381f2fa71a79ba483
was reverted in
https://github.com/macports/macports-ports/commit/ed352751afc52a036f329e7d34d418d367cf6cb5
I am not sure why the change was reverted, but currently `-stdlib=macports-libstdc++` does not work on Apple Silicon.

Reply:
TBH the whole variant that enables `macports-libstdc++` should only be active on the system that actually needs it. Currently its on by default across the board, which I also don't think is correct.

What systems actually need to be able to use `macports-libstdc++` ?

Reply:
> TBH the whole variant that enables `macports-libstdc++` should only be active on the system that actually needs it. Currently its on by default across the board, which I also don't think is correct.
> 
> What systems actually need to be able to use `macports-libstdc++` ?

Much has changed with the libc++ vs libstdc++ issue since `variant libstdcxx` was first created.
So it is entirely possible that the use of `macports-libstdc++` can be significantly reduced.
Default or not however, the variant is broken on Apple Silicon machines.